kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3Storage.java

```diff
@@ -179,6 +179,10 @@ public OutputStream create(String path, S3SinkConnectorConfig conf, boolean over
   }
 
   public S3OutputStream create(String path, boolean overwrite) {
+    return create(path, overwrite, false);
+  }
+
+  public S3OutputStream create(String path, boolean overwrite, boolean isParquetFormat) {
```

Comment:
I'm a bit torn about creating yet another `public` method. People actually use them and extend them, and then we need to deprecate the ones that are not useful after some time. 

Again the information we need to decide whether to use parquet is available in the config. 
How about using internally: 
`ParquetFormat.class.isAssignableFrom(this.conf.getClass(S3SinkConnectorConfig.FORMAT_CLASS_CONFIG))` in order to decide?

Reply:
It is a better choice. I did that because I hadn't found how to access config for format.class then. Thanks for your suggestion.